parlai/core/torch_ranker_agent.py

```diff
@@ -323,6 +351,8 @@ def report(self):
             m['loss'] = self.metrics['loss']
             m['mean_loss'] = self.metrics['loss'] / examples
             m['mean_rank'] = self.metrics['rank'] / examples
+            if self.metrics['train_accuracy'] > 0:
+                m['train_accuracy'] = self.metrics['train_accuracy'] / examples
```

Comment:
does it make more sense to guard this with self.opt rather than is 0? What if I literally get 0 accuracy lol.

Reply:
good point lol will update